packaging/build-docker-centos-rpm.sh

```diff
@@ -21,10 +21,10 @@ echo "Install needed packages: "
 docker exec -it $containerName sh -c "yum install -y git rpm-build redhat-rpm-config gcc-c++ make"
 
 echo "Install needed packages: "
-docker exec -it $containerName sh -c "curl --silent --location https://rpm.nodesource.com/setup_10.x | bash -"
+docker exec -it $containerName sh -c "curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash"
 
 echo "Install needed packages: "
-docker exec -it $containerName sh -c "yum install -y nodejs"
+docker exec -it $containerName sh -c "nvm install 10.15.0"
```

Comment:
This pins the version for the build container which is correct, but we also need to pin the required Node.js version in the RPM package itself. See https://github.com/bbc/speculate#node-versions.